app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsFragment.kt

```diff
@@ -179,7 +180,10 @@ class ChatRoomsFragment : Fragment(), ChatRoomsView {
     private fun subscribeUi() {
```

Comment:
This is unrelated to your PR but do you also find this function name incredibly non-intuitive? Shouldn't be chatRoomClicked or something like this?

Reply:
The essence of Subscribe UI here is that you're subscribing to all the changes/events in the UI. 
In it one thing you do is define the onClick handler for room click.